rewrite your-first-algo.md

lemonez opened the pull request:
Basically a re-write. Much shorter now, removed unnecessary info. Some screenshots could use updating; will replace the appropriate files in that directory.

lemonez committed:
rewrite your-firsts-algo.md

Basically a re-write. Some screenshots could use updating; will replace the appropriate files in that directory.

lemonez committed:
small fix in your-first-algo.md

lemonez committed:
delete duplicate header

bnlb commented on _pages/algorithm-development/your-first-algo.md:
```diff
@@ -13,216 +13,109 @@ redirect_from:
+On Algorithmia, every algorithm is required to have an `apply()` function—it defines the algorithm's entry point and serves to standardize algorithms across the platform regardless of programming language. Authors are encouraged to leverage this standardization and to design their algorithms in a way that makes them predictable for end users and that supports algorithm pipelining (chaining multip ...
 
-Under the **Versions** tab you can view information about all of the discreet algorithm builds that have occurred, including specific Git hashes and the version number associated with each.
+In order to run the algorithm, you'll first need to compile (build) it by clicking the "Build" button on the top-right corner of the source code editor. You'll need to complete this step even for an interpreted language such as Python, because we still need to package up all the source code into a callable algorithm instance on the back end.
```
+1 for describing why this needs to happen 👏

lemonez replied:
👍🏼

bnlb commented on _pages/algorithm-development/your-first-algo.md:
```diff
@@ -13,216 +13,109 @@ redirect_from:
+As an integration-first platform, we encourage you to connect to whatever data source(s) you're already using. We also enable you to host data directly on our platform for lower-latency access, and we even have several native integrationss with cloud-hosted data platforms.
 
-After you've finished this tutorial, you'll probably want to check out the <a href="{{site.baseurl}}/algorithm-development/languages">Language Guides</a> for how to write algorithms in the language you prefer, such as R, Python, Rust, Ruby, Java, Scala, or JavaScript.
+See our <a href="http://docs.algorithmia.com/#data">Data API</a> docs and our <a href="{{site.baseurl}}/data">Developer Center</a> docs to get started connecting to data sources. For a deeper dive tutorial, you can also check out our [Training Center](https://training.algorithmia.com/path/enterprise-20-2-data-scientist/enterprise-20-2-using-data-sources).
```
This is an old url, can avoid a couple of directs by hitting the new url directly
```suggestion
See our <a href="https://algorithmia.com/developers/api/#data">Data API</a> docs and our <a href="{{site.baseurl}}/data">Developer Center</a> docs to get started connecting to data sources. For a deeper dive tutorial, you can also check out our [Training Center](https://training.algorithmia.com/path/enterprise-20-2-data-scientist/enterprise-20-2-using-data-sources).
```

lemonez replied:
🙏🏼  thanks

bnlb reviewed (dismissed):
✍️

lemonez committed:
remove invalid link in your-first-algo.md

New Training Center rollout is blocked 🙄 not yet live so we can't link to it yet.
